.github/workflows/gradual-strict-null-checks.yml

```diff
@@ -50,9 +50,9 @@ jobs:
           YARN_1="yarn --mutex network --cwd ./current"
           YARN_2="yarn --mutex network --cwd ./main"
 
-          $YARN_1 install --ignore-scripts &> /dev/null && $YARN_1 build:backend --strictNullChecks true 2> .stderr-current > .out-current &
+          $YARN_1 install  &> /dev/null && $YARN_1 build:backend --strictNullChecks true 2> .stderr-current > .out-current &
```

Comment:
I think we can ignore this file as it's not working as intended... although we could test if running it with yarn 4 fixes it :thinking: